core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -201,33 +221,33 @@ contract Liquidatable is PricelessPositionManager {
                 sponsor: sponsor,
                 liquidator: msg.sender,
                 state: Status.PreDispute,
-                lockedCollateral: _getCollateral(positionToLiquidate.rawCollateral),
-                tokensOutstanding: positionToLiquidate.tokensOutstanding,
-                liquidatedCollateral: positionCollateral.sub(positionToLiquidate.withdrawalRequestAmount),
+                liquidationTime: getCurrentTime(),
+                tokensOutstanding: tokensToLiquidate,
+                lockedCollateral: lockedCollateral,
+                liquidatedCollateral: liquidatedCollateral,
                 rawUnitCollateral: _convertCollateral(FixedPoint.fromUnscaledUint(1)),
                 disputer: address(0),
-                liquidationTime: getCurrentTime(),
                 settlementPrice: FixedPoint.fromUnscaledUint(0)
             })
         );
 
         // Add to the global liquidation collateral count.
-        _addCollateral(rawLiquidationCollateral, positionCollateral);
+        _addCollateral(rawLiquidationCollateral, lockedCollateral);
```

Comment:
I still think this should go after `_reduceSponsorPosition` just to be safe and make sure that contract never, even briefly, thinks it has more collateral than it does

Reply:
I don't have a preference either way, we ca definitely go with after reducing position.

Reply:
Done.